src/debug/dotNetDebugProvider.ts

```diff
@@ -46,6 +47,16 @@ export class DotNetDebugProvider implements IDebugProvider {
                 quoteArgs: false
             }
         };
+        const map = extensionConfig.getDotnetDebugSourceFileMap();
+        if (map) {
+            try{
+                const json: string = `{"${map.replace("\\", "\\\\")}":"${workspaceFolder.replace("\\", "\\\\")}"}`;
+                const sourceFileMap: JSON = JSON.parse(json);
+                debugConfiguration['sourceFileMap'] = sourceFileMap;
+            }catch(Error){
```

Comment:
Nitpick: separate braces with spaces 
```suggestion
            } catch(Error) {
```